cogs/help.py

```diff
@@ -440,6 +391,31 @@ async def transfer(self, ctx, *, new_author: Member):
         )
         await ctx.guild.get_channel(HELP_LOGS_CHANNEL_ID).send(embed=embed_log)
 
+    @slash_command(guild_ids=[GUILD_ID])
+    @application_checks_has_role(HELP_MOD_ID)
+    async def topic(
+        self,
+        interaction: Interaction,
+        *,
+        topic: str = SlashOption(
```

Comment:
Likely this should be required?

Reply:
They are both required?

Reply:
![image](https://user-images.githubusercontent.com/34910574/169332945-3806cad5-bf07-4a7f-a32d-9f05d8cbf1f9.png)

Reply:
They are both required as seen here: ![yeet](https://v.adiscorduser.com/RO7TzFy.mp4)